docarray/documents/image.py

```diff
@@ -92,10 +95,25 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[ImageUrl]
-    tensor: Optional[ImageTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[ImageBytes]
+    url: Optional[ImageUrl] = Field(
+        description='URL of the remote image',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/image-data/apple.png?raw=true',
+    )
+    tensor: Optional[ImageTensor] = Field(
+        description='''Tensor object of the image which can be specifed to one of
+    `ImageNdArray`, `ImageTorchTensor`, `ImageTensorflowTensor`.''',
+        example='img.url.load()',
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='''Embedding field is used to store tensor objects of type
+        Tensorflow, PyTorch, NumPy and Jax''',
+        example='np.zeros((3, 32, 32))',
+    )
+    bytes_: Optional[ImageBytes] = Field(
+        description='''Bytes object of the image which is an instance of
+        `ImageBytes.''',
+        example='img.url.load_bytes()',
```

Comment:
I do not think is a good example, I would just remove it

Reply:
Removed example for `bytes_`